# Provide an Obstacle detction with the depth camera

# position of the obstacle
uint8 LEFT	= 0
uint8 CENTER	= 1
uint8 RIGHT	= 2
uint8 FRONT	= 3
uint8 NONE	= 4

uint8 position
